lib/sidekiq/client.rb

```diff
@@ -138,6 +141,17 @@ def push_bulk(items)
       jid = items.delete("jid")
       raise ArgumentError, "Explicitly passing 'jid' when pushing more than one job is not supported" if jid && args.size > 1
 
+      spread_interval = items.delete(:spread_interval) || items.delete("spread_interval")
+      raise ArgumentError, "Jobs 'spread_interval' must be a positive Numeric" if spread_interval && (!spread_interval.is_a?(Numeric) || spread_interval <= 0)
+      raise ArgumentError, "Only one of 'at' or 'spread_interval' can be provided" if at && spread_interval
+
+      if !at && spread_interval
+        # Do not use spread interval smaller than pooling interval.
+        spread_interval = [spread_interval, 5].max
```

Comment:
@fatkodima Should there be an explicit `to_i` or `to_f` here, to be consistent with the implementation of `perform_at`?

Right now, I don't dare to pass an `ActiveSupport::Duration`, even though it will probably work. 😉